src/Commands/RepositoryGeneratorCommand.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Isneezy\Celeiro\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Filesystem\Filesystem;
+use Illuminate\Support\Str;
+
+
+class RepositoryGeneratorCommand extends Command
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $signature = 'make:repository {repo}{--model=null}';
+
+    /**
+     * @var string
+     */
+    protected $description = "Generate a new repository given a Model name";
+
+
+    /**
+     * The filesystem instance.
+     *
+     * @var \Illuminate\Filesystem\Filesystem
+     */
+    protected $files;
+
+
+    public function __construct(Filesystem $files)
+    {
+        parent::__construct();
+        $this->files = $files;
+    }
+
+
+    public function handle()
+    {
+        $repositoryName = ucfirst($this->argument('repo'));
```

Comment:
Think here we can use camelcase and then ucfirst
Ive not tested yet but
foo_bar => FooBar
foobar => Foobar
fooBar => FooBar